includes/class-convertkit-cache-plugins.php

```diff
@@ -144,6 +148,26 @@ public function wp_rocket_disable_caching_and_minification_on_landing_pages() {
 
 	}
 
+	/**
+	 * Disable "Delay JavaScript execution" on ConvertKit scripts when the WP Rocket Plugin is installed, active
```

Comment:
I'm a bit confused by this comment - it seems to imply that this is added conditionally based on whether the WP Rocket plugin is install, active, and set to delay JS execution, but I don't see where that check is actually happening.

Reply:
Fixed, thanks!